src-java/server42/server42-stats-messaging/src/main/java/org/openkilda/server42/stats/messaging/flowrtt/Statistics.java

```diff
@@ -1,5 +1,5 @@
 // Generated by the protocol buffer compiler.  DO NOT EDIT!
-// source: server42/statistics.proto
+// source: statistics.proto
```

Comment:
Should it be removed from repo now?

Reply:
We need to discuss that. I will do it with a separate PR if somebody needs that.